docs/content/v0.20/examples.md

```diff
@@ -25,7 +27,7 @@ The following steps establish an initial state used in the examples below.
    ```
 
 3. Create an inventory & mailbox space of type `vcluster` running *OCM* (Open Cluster Management)
-in KubeFlex. Note that `-p ocm` runs a post-create hook on the *vcluster* control plane 
+in KubeFlex. Note that `-p ocm` runs a post-create hook on the *vcluster* control plane
```

Comment:
Thanks for finding and fixing all these extra spaces at end of line.
If these fixes were in a separate PR then it could be merge without any debate.